nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -14,6 +14,20 @@ These release notes track the development of the {run-once-operator} for {produc
 
 For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
 
+[id="run-once-duration-override-operator-release-notes-1-1-2"]
+== Run Once Duration Override Operator 1.1.2
```

Comment:
You should consider using the attribute for the operator name for these instances throughout the file too.